llm-servers/sbert/gpu/app.py

```diff
@@ -32,7 +32,7 @@ async def process_strings(input_array: StringArray):
 parser.add_argument('--model_path', type=str, required=False, default='/mnt/models', help='Path to the Sentence Transformer model')
 args = parser.parse_args()
 
-model = SentenceTransformer(args.model_path)
+model = SentenceTransformer(args.model_path, trust_remote_code=True)
```

Comment:
Can you put the trust_remote_code as an argument, with a default value as False?
Enabling it should be done only voluntarily when you know you need it.

Reply:
Sure, I'll do that